config/routes.rb

```diff
@@ -39,6 +39,9 @@
         member do
           post :cancel
         end
+        collection do
+          match '/bulk/destroy' => :bulk_destroy, :via => [:post]
```

Comment:
How about we make the path match what we do for [other bulk actions](https://github.com/theforeman/foreman-tasks/blob/master/config/routes.rb#L51) (`bulk_destroy` vs `bulk/destroy`). Then you should be able to do something like
```ruby
collection do
  post :bulk_destroy
end
```

Reply:
I was taking inspiration from hosts but this makes more sense, thanks!